lib/ansible/modules/cloud/ovirt/ovirt_host_networks.py

```diff
@@ -160,48 +174,48 @@
 )
 
 
-def get_mode_type(mode_number):
-    """
-    Adaptive transmit load balancing (balance-tlb): mode=1 miimon=100
-    Dynamic link aggregation (802.3ad): mode=2 miimon=100
-    Load balance (balance-xor): mode=3 miimon=100
-    Active-Backup: mode=4 miimon=100 xmit_hash_policy=2
-    """
+def get_bond_options(mode, opts_str):
     options = []
-    if mode_number is None:
-        return options
 
     def get_type_name(mode_number):
         """
         We need to maintain this type strings, for the __compare_options method,
         for easier comparision.
         """
-        return [
+        modes = [
             'Active-Backup',
             'Load balance (balance-xor)',
             None,
             'Dynamic link aggregation (802.3ad)',
-        ][mode_number]
-
-    try:
-        mode_number = int(mode_number)
-        if mode_number >= 1 and mode_number <= 4:
-            if mode_number == 4:
-                options.append(otypes.Option(name='xmit_hash_policy', value='2'))
```

Comment:
Why did you this? I think you have to pass it, because those are deafult options for bond modes 1-4.

Reply:
I will check, but I am pretty sure default values will be set even if they're not explicitly specified (being default)

Reply:
I was wrong, I'll re-add this